lib/widgets/content.dart

```diff
@@ -460,8 +460,7 @@ class _InlineContentBuilder {
       return WidgetSpan(alignment: PlaceholderAlignment.middle,
         child: UserMention(node: node));
     } else if (node is UnicodeEmojiNode) {
-      return WidgetSpan(alignment: PlaceholderAlignment.middle,
-        child: MessageUnicodeEmoji(node: node));
+      return TextSpan(text: node.emojiUnicode, recognizer: _recognizer);
```

Comment:
nit in commit message:

    - Generate a tappable TextSpan for each unicode emoji

It's true that this will be tappable if it's inside something tappable (i.e., a link). But this line in the commit message sounds like it's saying the span will be tappable in its *own* way, related somehow to its status as an emoji. Which is a plausible feature — you could imagine tapping (maybe more likely long-tapping, but possibly just tapping) to pull up the name of the emoji in a tooltip, or something like that.

So just s/tappable // clarifies it, I think.